### ISSUE TITLE
Dialog boxes are too small on Samsung Galaxy S2

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#242]
```
When running the game on Samsung Galaxy S2 using Android 2.3.3, most of the 
dialog boxes are displayed with a width that is too small.

The dialog boxes should display as the full screen width.

This is mostly visible on the monsterinfo, iteminfo and monsterencounter dialog 
boxes, but also on the bulkselection interface.
```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 9 Sep 2011 at 6:17

**Comment by GoogleCodeExporter** [comment]
```
Some troubleshooting reveals that setting layout_width="match_parent" on all 
views in the dialogs is simply not sufficient, unless there is something that I 
am missing.

A work-around is to add an empty listview, since that expands the layouts to 
their full width. Something like this:
<ListView android:layout_width="match_parent" 
android:layout_height="wrap_content" />

```

Original comment by `oskar.wi...@gmail.com` on 9 Sep 2011 at 6:19

**Comment by GoogleCodeExporter** [comment]
```
This should be resolved in r154.

Originally reported here:
http://andors.techby2guys.com/viewtopic.php?f=3&t=979
```

Original comment by `oskar.wi...@gmail.com` on 9 Sep 2011 at 7:06

**Comment by GoogleCodeExporter** [comment]
```
Fix has been confirmed by submitter. Marking issue as ready for v0.6.10.
```

Original comment by `oskar.wi...@gmail.com` on 12 Sep 2011 at 1:53
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
